### ISSUE TITLE
[dataio] break dataio-pyshovel into multiple files (Trac #1112)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1106]
It's getting a little large for a single file, so break it up.

Migrated from https://code.icecube.wisc.edu/ticket/1112
```json
{
    "status": "closed", 
    "changetime": "2017-10-12T18:17:09", 
    "description": "It's getting a little large for a single file, so break it up.", 
    "reporter": "david.schultz", 
    "cc": "", 
    "resolution": "wontfix", 
    "_ts": "1507832229295176", 
    "component": "combo core", 
    "summary": "[dataio] break dataio-pyshovel into multiple files", 
    "priority": "normal", 
    "keywords": "", 
    "time": "2015-08-12T21:24:57", 
    "milestone": "Long-Term Future", 
    "owner": "david.schultz", 
    "type": "enhancement"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20151204T16:57:54: 
* david.schultz commented:

> Do this after #1423.


* david.schultz changed milestone from "offline V15-11-00" to "April 2016"
* david.schultz changed priority from "major" to "normal"

**Comment by icecube-trac** [comment]
Trac update at 20160726T16:31:25: david.schultz changed milestone from "May 2016" to "November 2016"

**Comment by icecube-trac** [comment]
Trac update at 20161111T19:21:04: 
* david.schultz changed cc from "hdembinski" to ""
* david.schultz changed milestone from "November 2016" to "Long-Term Future"

**Comment by icecube-trac** [comment]
Trac update at 20171012T18:17:09: 
* david.schultz commented:

> Pursuant to #2046, dataio-pyshovel will be retired.


* david.schultz changed status from "new" to "closed"
* david.schultz changed resolution from "" to "wontfix"
